x-pack/plugins/cases/public/components/all_cases/all_cases_list.test.tsx

```diff
@@ -315,28 +310,26 @@ describe('AllCasesListGeneric', () => {
   it('renders the columns correctly', async () => {
     appMockRenderer.render(<AllCasesList isSelectorView={false} />);
 
-    const casesTable = within(screen.getByTestId('cases-table'));
-
-    expect(casesTable.getByTitle('Name')).toBeInTheDocument();
-    expect(casesTable.getByTitle('Category')).toBeInTheDocument();
-    expect(casesTable.getByTitle('Created on')).toBeInTheDocument();
-    expect(casesTable.getByTitle('Updated on')).toBeInTheDocument();
-    expect(casesTable.getByTitle('Status')).toBeInTheDocument();
-    expect(casesTable.getByTitle('Severity')).toBeInTheDocument();
-    expect(casesTable.getByTitle('Tags')).toBeInTheDocument();
-    expect(casesTable.getByTitle('Alerts')).toBeInTheDocument();
-    expect(casesTable.getByTitle('Comments')).toBeInTheDocument();
-    expect(casesTable.getByTitle('External incident')).toBeInTheDocument();
-    expect(casesTable.getByTitle('Actions')).toBeInTheDocument();
+    const casesTable = within(await screen.findByTestId('cases-table'));
+
+    expect(await casesTable.findByTitle('Name')).toBeInTheDocument();
```

Comment:
Do you think maybe it would be faster to do `const headers = findAll` once and then `expect(headers[I]).toHaveTextContent('whatever')` ?

Reply:
it's a good point but the title is just a span with one class, so there is nothing which could be used for `findAll`.

Tried to use `await casesTable.findAllByRole('columnheader')` but then we will have to go into each header's child nodes. 

Also I think in this approach we restrict the test with particular sequence of columns.